google-oauth-client/src/main/java/com/google/api/client/auth/openidconnect/IdTokenVerifier.java

```diff
@@ -231,8 +234,10 @@ public final Collection<String> getAudience() {
    *
    * @param idToken ID token
    * @return {@code true} if verified successfully or {@code false} if failed
+   * @throws IOException thrown if verification failed to run. For example, failure to get public
```

Comment:
nit: consider editing to say `@throws IOException if verification failed to run.`